contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3551,6 +3603,9 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 					GrantStmt  *stmt;
 					PlannedStmt *wrapper;
 					RoleSpec *rolspec = create_schema->authrole;
+					Oid       owner_oid = InvalidOid;
+					Oid       db_accessadmin = get_role_oid(get_db_accessadmin_role_name(get_cur_db_name()), true);
```

Comment:
Missing_ok should be false.

Reply:
Problem here is during upgrade from 3.0 -> 3.1 we create guest schemas for all existing babelfish databases. 
During this CREATE SCHEMA operation db_accessadmin will not exists.
`create_guest_schema_for_all_dbs` <-- Problematic function name

Should I instead reuse this query string `(CREATE LOGICAL DATABASE )` in `create_guest_schema_for_all_dbs` ?
Then we can change missing okay to false.

Reply:
Make some change and setting missing ok to false now